From 32a58a8676a9c32787d02dcd415d2433f595e1f0 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Fri, 19 Oct 2018 17:00:17 +0300
Subject: [PATCH 683/767] staging: fsl-dpaa2/mac: Fix uninitialized variable

Reported by coverity.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index 5288e9a6b2ac..9ea2dd674588 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -355,7 +355,7 @@ static irqreturn_t dpaa2_mac_irq_handler(int irq_num, void *arg)
 	struct device *dev = (struct device *)arg;
 	struct fsl_mc_device *mc_dev = to_fsl_mc_device(dev);
 	struct dpaa2_mac_priv *priv = dev_get_drvdata(dev);
-	struct dpmac_link_cfg link_cfg;
+	struct dpmac_link_cfg link_cfg = { 0 };
 	u32 status;
 	int err;
 
-- 
2.17.0

